pages/common/bc.md

```diff
@@ -1,20 +1,29 @@
 # bc
 
 > An arbitrary precision calculator language.
-> More information: <https://manned.org/bc>.
+> Get help or version: bc --help|--version.
```

Comment:
the page has 6 examples, add `--help` and `--version` as examples at the end.
```suggestion
```

Reply:
But in general case there is no always enough space for such `--help` and `--version` examples. That's why I moved them to the top to mention them always and not to pollute code samples. :) I don't want miss smth more important due `--help` and `--version` examples and I wanna provide info about how view help for user.

Reply:
moving `--help/--version` to the command description, it's giving to much importance to how to get the `--version` of the program.

Reply:
Yes, that's true. :) But on the other hand there are too many tool versions, for instance `mawk`, `gawk` and others. And user have to know how to ensure what command he/she uses. 🤔

Reply:
I'm with @navarroaxel , the purpose of top lines are brief description of a command. Put example here will break tldr's formatting.

Reply:
I would argue that `--help` and `--version` are not suitable for inclusion in a tldr page at all, as they are pretty much a standard convention.